test_compare_taxa_summaries.py randomly fails

```
======================================================================
FAIL: test_compute_correlation_paired (__main__.CompareTaxaSummariesTests)
Test runs correctly on standard input taxa summaries.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/workspace/qiime-github-pr/qiime_software/qiime-1.6.0-repository-5bfb37d7/tests/test_compare_taxa_summaries.py", line 923, in test_compute_correlation_paired
    self.assertTrue(found_match)
AssertionError: False is not true

----------------------------------------------------------------------
Ran 38 tests in 8.484s

FAILED (failures=1)
```

--- a/tests/test_compare_taxa_summaries.py
+++ b/tests/test_compare_taxa_summaries.py
@@ -41,7 +41,7 @@ class CompareTaxaSummariesTests(TestCase):
     def setUp(self):
         """Define some sample data that will be used by the tests."""
         # How many times to test a stochastic p-value.
-        self.p_val_tests = 10
+        self.p_val_tests = 20
 
         self.taxa_summary1 = (['Even1','Even2','Even3'],
             ['Bacteria;Actinobacteria;Actinobacteria(class);Actinobacteridae',

